ui/page/settings_page.go

```diff
@@ -92,11 +90,6 @@ func NewSettingsPage(l *load.Load) *SettingsPage {
 
 	pg.backButton, pg.infoButton = components.SubpageHeaderButtons(l)
 
-	// pg.peerLabel = l.Theme.Body1("")
-	// pg.peerLabel.Color = l.Theme.Color.GrayText2
-
-	// pg.agentLabel = l.Theme.Body1("")
-	// pg.agentLabel.Color = l.Theme.Color.GrayText2
```

Comment:
remove comments

Reply:
There are no commented codes... these was actually removed not added.

Reply:
Oh, my bad